assets/stylesheets/index.scss

```
@@ -6,16 +6,19 @@
 // Base
 @import 'base/button';
 @import 'base/form';
-@import 'base/nav';
 @import 'base/table';
 @import 'base/typography';
 
 // Components
 @import 'components/alert';
+@import 'components/search-result';
+@import 'components/search-result-list';
```

Since the class name is `.list-search-result`, teh file name must be:

```suggestion
@import 'components/list-search-result';
```

---

Fixed in f8bb9f26308ae2bbe6c5ec94a948f49bea25862b